### PR TITLE
update create acc guide

### PR DESCRIPTION
**Pull request by konopkja**
making it easier to read for beginners, some content was inadequate and excessive, also image needed to be updated after find wallet paage redesign

### CODE REVIEW

File: public/content/guides/how-to-create-an-ethereum-account/index.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ lang: en
...
+**Anyone can create an Ethereum account for free.** You just need to install a crypto wallet app. Wallets create and manage your Ethereum account. They can send transactions, check your balances and connect you to other apps built on Ethereum.
 
-Unlike opening a new account with a company, creating an Ethereum account is done freely, privately and without requiring permission. Accounts are controlled by keys that your wallet software helps you create, and are not issued by a third party, nor stored in a central registry.
+With a wallet you can also log into any token exchange, games, [NFT](/glossary/#nft) marketplaces instantly. There is no need for individual registration, one account is shared for all apps built on Ethereum.
```
Comment (wackerow) [inline public/content/guides/how-to-create-an-ethereum-account/index.md:11]: :heart:

File: public/content/guides/how-to-create-an-ethereum-account/index.md [hunk 2]
```diff
@@ -30,43 +30,43 @@ Once you have decided on a specific wallet, visit their official website or app
...
+The first time you open your new wallet you might be asked to choose between creating a new account or importing an existing one. Click on the new account creation. **This is the step during which the wallet software generates your Ethereum account.**
 
 ## Step 4: Store your recovery phrase
 
-Some apps will request you to save a secret 'seed phrase' (you might also see this referred to as a "recovery phrase" or a "mnemonic"). Keeping this seed phrase safe is extremely important! The seed phrase is used to generate a secret key for an account which can be used to sign and send transactions. Any person who knows the seed phrase can take control of all the accounts generated by it. Never share the seed phrase with anyone. The seed phrase should contain 12 to 24 randomly generated words (the order of the words matters).
+Some apps will request you to save a secret 'recovery phrase' (sometimes called a "seed phrase" or a "mnemonic"). Keeping this phrase safe is extremely important! This is used to generate your Ethereum account and can be used to submit transactions.
```
Comment (wackerow) [inline public/content/guides/how-to-create-an-ethereum-account/index.md:37]: Any reason we switchd from single-quotes to double-quotes mid-sentence?

```suggestion
Some apps will request you to save a secret "recovery phrase" (sometimes called a "seed phrase" or a "mnemonic"). Keeping this phrase safe is extremely important! This is used to generate your Ethereum account and can be used to submit transactions.
```

File: public/content/guides/how-to-create-an-ethereum-account/index.md [hunk 3]
```diff
@@ -30,43 +30,43 @@ Once you have decided on a specific wallet, visit their official website or app
...
 
 ### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
 
-No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
+No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
```
Comment (wackerow) [inline public/content/guides/how-to-create-an-ethereum-account/index.md:60]: An aside, would suggest we just follow the capitalization used on the bitcoin.com website for WBTC: "Wrapped Bitcoin"

```suggestion
No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped Bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
```

**Review by wackerow — APPROVED** [review]
Looks good @konopkja! Just two tiny suggestions, but don't see them as blocking so going ahead with approval.

**Review by corwintines — APPROVED** [review]

**Pull request merged**